contrib/for-tests/network-tester/slow-rc.json

```diff
@@ -0,0 +1,42 @@
+{
+  "kind": "ReplicationController",
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "slow-rc",
```

Comment:
Another rebase snafu.

I think you want:
https://github.com/kubernetes/kubernetes/blob/master/contrib/for-tests/network-tester/rc.json